general-info/accessibility.html

```diff
@@ -133,25 +135,61 @@ <h3>Using a Microphone</h3>
 
 	<div class="row">
 		<div class="col-12">
-			<h2 id="transportation">Public Transportation</h2>
+			<h2 id="venue">Conference Venues</h3>
+		
+			<h3>Frist Campus Center (Main Conference / Game Night)</h3>
+			<p>The main conference will be held on the Princeton campus in the <a href="https://www.princeton.edu/frist/">Frist Campus Center</a> on the <a href="https://www.princeton.edu/frist/floorplans/floorplansBc.html">B Level</a>. Elevators and accessible bathrooms are located on this level outside of the conference hall. 
+			</p>
+			
+			<p>There is an accessible bus stop that is located right outside of the garage and lets riders off on Washington Road on the North side of Frist Campus Center on the opposite side of the road.  Participants can navigate on the walkway heading south towards Frist and enter the building either on the backside, using a ramp then chair lift leaving them on the first floor or move along the side of the building to an entrance with an automatic door opener onto the first floor.  There is an elevator on the opposite side of the building that leads to the B level and conference location.</p>
+			
+			<h3>Julis Romo Rabinowitz Building (Pre-Conference Workshops)</h3>
+			<p>More information about access forthcoming.</p>
+			
+			<h3>Prospect House (Lunch)</h3>
+			<p>Daily lunch will be served at <a href="https://www.princeton.edu/prospecthouse/index.html">Prospect House</a>. While the main entrance has two steps, but there is another door with no steps, also on the front of the building. An accessible bathroom is upstairs. The elevator is just inside the kitchen door.</p>
+			<p>It is worth noting that the walking path between the Frist Center and Prospect House can be steep at points. Please contact  <p>Contact <a href="mailto:{{site.data.conf.accessibility-email}}">{{site.data.conf.accessibility-email}}</a> to request any assistance. Food is also available for purchase in Frist.</p>
+
+			<h3>Palmer House (Reception)</h3>
+			<p>The Wednesday night reception will be held in <a href="https://www.princeton.edu/palmerhouse/">. Further details about access are forthcoming.</p>
```

Comment:
I think you need to add the text and closing `</a>` tag here.

Reply:
Yep. Fixed